[2024-05-10T19:39:38.644+0000] {processor.py:161} INFO - Started process (PID=29) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:39:38.645+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:39:38.654+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:39:38.648+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:39:38.659+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:39:38.682+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.053 seconds
[2024-05-10T19:40:04.809+0000] {processor.py:161} INFO - Started process (PID=30) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:40:04.809+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:40:04.811+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:40:04.811+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:40:04.815+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:40:04.831+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.028 seconds
[2024-05-10T19:40:34.947+0000] {processor.py:161} INFO - Started process (PID=31) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:40:34.948+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:40:34.950+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:40:34.950+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:40:34.959+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:40:34.956+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 6
    sys.path.insert(__index:0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
                           ^
SyntaxError: invalid syntax
[2024-05-10T19:40:34.959+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:40:34.980+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.038 seconds
[2024-05-10T19:41:05.084+0000] {processor.py:161} INFO - Started process (PID=32) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:41:05.085+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:41:05.087+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:41:05.087+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:41:05.091+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:41:05.090+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 6
    sys.path.insert(__index:0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
                           ^
SyntaxError: invalid syntax
[2024-05-10T19:41:05.091+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:41:05.110+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.032 seconds
[2024-05-10T19:41:35.229+0000] {processor.py:161} INFO - Started process (PID=33) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:41:35.230+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:41:35.232+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:41:35.232+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:41:35.236+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:41:35.236+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 6
    sys.path.insert(__index:0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
                           ^
SyntaxError: invalid syntax
[2024-05-10T19:41:35.237+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:41:35.257+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.034 seconds
[2024-05-10T19:42:05.367+0000] {processor.py:161} INFO - Started process (PID=34) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:42:05.369+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:42:05.371+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:42:05.371+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:42:05.378+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:42:05.377+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 6
    sys.path.insert(__index:0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
                           ^
SyntaxError: invalid syntax
[2024-05-10T19:42:05.379+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:42:05.399+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.039 seconds
[2024-05-10T19:42:10.404+0000] {processor.py:161} INFO - Started process (PID=35) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:42:10.406+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:42:10.408+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:42:10.408+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:42:10.413+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:42:10.432+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.036 seconds
[2024-05-10T19:42:35.549+0000] {processor.py:161} INFO - Started process (PID=36) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:42:35.550+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:42:35.554+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:42:35.552+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:42:35.560+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:42:35.578+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.035 seconds
[2024-05-10T19:43:05.684+0000] {processor.py:161} INFO - Started process (PID=37) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:43:05.685+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:43:05.688+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:43:05.687+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:43:05.691+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:43:05.704+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.027 seconds
[2024-05-10T19:43:20.760+0000] {processor.py:161} INFO - Started process (PID=38) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:43:20.761+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:43:20.764+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:43:20.763+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:43:20.769+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:43:20.767+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 13, in <module>
    file_
NameError: name 'file_' is not defined
[2024-05-10T19:43:20.769+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:43:20.793+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.039 seconds
[2024-05-10T19:43:50.914+0000] {processor.py:161} INFO - Started process (PID=39) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:43:50.916+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:43:50.919+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:43:50.918+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:43:50.927+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:43:50.924+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 13, in <module>
    file_
NameError: name 'file_' is not defined
[2024-05-10T19:43:50.928+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:43:50.961+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.058 seconds
[2024-05-10T19:44:21.071+0000] {processor.py:161} INFO - Started process (PID=40) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:44:21.073+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:44:21.075+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:44:21.075+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:44:21.080+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:44:21.078+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 13, in <module>
    file_
NameError: name 'file_' is not defined
[2024-05-10T19:44:21.080+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:44:21.100+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.037 seconds
[2024-05-10T19:44:51.234+0000] {processor.py:161} INFO - Started process (PID=41) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:44:51.235+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:44:51.237+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:44:51.237+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:44:51.241+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:44:51.240+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 13, in <module>
    file_
NameError: name 'file_' is not defined
[2024-05-10T19:44:51.242+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:44:51.261+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.034 seconds
[2024-05-10T19:45:21.377+0000] {processor.py:161} INFO - Started process (PID=42) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:45:21.378+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:45:21.381+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:45:21.380+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:45:21.385+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:45:21.384+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 13, in <module>
    file_
NameError: name 'file_' is not defined
[2024-05-10T19:45:21.385+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:45:21.404+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.033 seconds
[2024-05-10T19:45:51.546+0000] {processor.py:161} INFO - Started process (PID=43) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:45:51.547+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:45:51.549+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:45:51.549+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:45:51.554+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:45:51.553+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 13, in <module>
    file_
NameError: name 'file_' is not defined
[2024-05-10T19:45:51.554+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:45:51.572+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.033 seconds
[2024-05-10T19:46:21.681+0000] {processor.py:161} INFO - Started process (PID=44) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:46:21.682+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:46:21.685+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:46:21.685+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:46:21.689+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:46:21.688+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 13, in <module>
    file_
NameError: name 'file_' is not defined
[2024-05-10T19:46:21.690+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:46:21.707+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.032 seconds
[2024-05-10T19:46:34.752+0000] {processor.py:161} INFO - Started process (PID=45) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:46:34.753+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:46:34.756+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:46:34.756+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:46:34.767+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:46:34.766+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 24, in <module>
    extract = PythonOperator(
NameError: name 'PythonOperator' is not defined
[2024-05-10T19:46:34.768+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:46:34.791+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.045 seconds
[2024-05-10T19:46:49.865+0000] {processor.py:161} INFO - Started process (PID=46) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:46:49.868+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:46:49.872+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:46:49.871+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:46:49.890+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:46:49.889+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:46:49.892+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:46:49.931+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.079 seconds
[2024-05-10T19:47:20.000+0000] {processor.py:161} INFO - Started process (PID=47) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:47:20.001+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:47:20.003+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:47:20.003+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:47:20.011+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:47:20.009+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:47:20.011+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:47:20.032+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.040 seconds
[2024-05-10T19:47:50.145+0000] {processor.py:161} INFO - Started process (PID=48) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:47:50.146+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:47:50.149+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:47:50.148+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:47:50.156+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:47:50.155+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:47:50.156+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:47:50.175+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.038 seconds
[2024-05-10T19:48:20.284+0000] {processor.py:161} INFO - Started process (PID=49) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:48:20.285+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:48:20.287+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:48:20.287+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:48:20.296+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:48:20.294+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:48:20.296+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:48:20.315+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.036 seconds
[2024-05-10T19:48:50.422+0000] {processor.py:161} INFO - Started process (PID=50) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:48:50.423+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:48:50.426+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:48:50.425+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:48:50.435+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:48:50.434+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:48:50.435+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:48:50.457+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.043 seconds
[2024-05-10T19:49:20.564+0000] {processor.py:161} INFO - Started process (PID=51) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:49:20.565+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:49:20.568+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:49:20.567+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:49:20.577+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:49:20.576+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:49:20.578+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:49:20.597+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.040 seconds
[2024-05-10T19:49:50.725+0000] {processor.py:161} INFO - Started process (PID=52) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:49:50.726+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:49:50.729+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:49:50.728+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:49:50.735+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:49:50.734+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:49:50.735+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:49:50.753+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.037 seconds
[2024-05-10T19:50:20.869+0000] {processor.py:161} INFO - Started process (PID=53) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:50:20.870+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:50:20.873+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:50:20.872+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:50:20.880+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:50:20.879+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:50:20.881+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:50:20.901+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.042 seconds
[2024-05-10T19:50:51.011+0000] {processor.py:161} INFO - Started process (PID=54) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:50:51.012+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:50:51.014+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:50:51.014+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:50:51.023+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:50:51.021+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:50:51.024+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:50:51.044+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.042 seconds
[2024-05-10T19:51:21.145+0000] {processor.py:161} INFO - Started process (PID=55) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:51:21.146+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:51:21.148+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:51:21.148+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:51:21.155+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:51:21.154+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:51:21.155+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:51:21.172+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.032 seconds
[2024-05-10T19:51:51.274+0000] {processor.py:161} INFO - Started process (PID=56) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:51:51.275+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:51:51.277+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:51:51.277+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:51:51.285+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:51:51.284+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:51:51.285+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:51:51.301+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.032 seconds
[2024-05-10T19:52:21.403+0000] {processor.py:161} INFO - Started process (PID=57) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:52:21.404+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:52:21.406+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:52:21.406+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:52:21.413+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:52:21.412+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:52:21.414+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:52:21.431+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.033 seconds
[2024-05-10T19:52:51.541+0000] {processor.py:161} INFO - Started process (PID=58) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:52:51.542+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:52:51.544+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:52:51.544+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:52:51.553+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:52:51.552+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:52:51.554+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:52:51.572+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.037 seconds
[2024-05-10T19:53:21.670+0000] {processor.py:161} INFO - Started process (PID=59) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:53:21.671+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:53:21.673+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:53:21.673+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:53:21.680+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:53:21.679+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:53:21.680+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:53:21.698+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.034 seconds
[2024-05-10T19:53:51.808+0000] {processor.py:161} INFO - Started process (PID=60) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:53:51.809+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:53:51.811+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:53:51.810+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:53:51.817+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:53:51.817+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:53:51.818+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:53:51.837+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.035 seconds
[2024-05-10T19:54:21.938+0000] {processor.py:161} INFO - Started process (PID=61) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:54:21.939+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:54:21.942+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:54:21.941+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:54:21.948+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:54:21.947+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:54:21.949+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:54:21.966+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.033 seconds
[2024-05-10T19:54:52.079+0000] {processor.py:161} INFO - Started process (PID=62) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:54:52.080+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:54:52.082+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:54:52.082+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:54:52.089+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:54:52.088+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:54:52.089+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:54:52.108+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.034 seconds
[2024-05-10T19:55:22.209+0000] {processor.py:161} INFO - Started process (PID=63) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:55:22.210+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:55:22.212+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:55:22.212+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:55:22.219+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:55:22.218+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:55:22.220+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:55:22.239+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.035 seconds
[2024-05-10T19:55:52.343+0000] {processor.py:161} INFO - Started process (PID=64) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:55:52.344+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:55:52.346+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:55:52.346+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:55:52.353+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:55:52.352+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:55:52.354+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:55:52.373+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.037 seconds
[2024-05-10T19:56:22.470+0000] {processor.py:161} INFO - Started process (PID=65) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:56:22.471+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:56:22.473+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:56:22.472+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:56:22.479+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:56:22.478+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:56:22.480+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:56:22.496+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.032 seconds
[2024-05-10T19:56:52.618+0000] {processor.py:161} INFO - Started process (PID=66) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:56:52.619+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:56:52.622+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:56:52.621+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:56:52.630+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:56:52.629+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:56:52.631+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:56:52.649+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.039 seconds
[2024-05-10T19:57:22.760+0000] {processor.py:161} INFO - Started process (PID=67) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:57:22.761+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:57:22.763+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:57:22.763+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:57:22.771+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:57:22.771+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:57:22.772+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:57:22.798+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.046 seconds
[2024-05-10T19:57:52.909+0000] {processor.py:161} INFO - Started process (PID=68) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:57:52.910+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:57:52.913+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:57:52.912+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:57:52.919+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:57:52.918+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:57:52.920+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:57:52.937+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.035 seconds
[2024-05-10T19:58:23.042+0000] {processor.py:161} INFO - Started process (PID=69) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:58:23.043+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:58:23.046+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:58:23.046+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:58:23.054+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:58:23.053+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:58:23.055+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:58:23.075+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.039 seconds
[2024-05-10T19:58:53.189+0000] {processor.py:161} INFO - Started process (PID=70) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:58:53.190+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:58:53.193+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:58:53.192+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:58:53.201+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:58:53.200+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:58:53.201+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:58:53.222+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.039 seconds
[2024-05-10T19:59:23.335+0000] {processor.py:161} INFO - Started process (PID=71) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:59:23.336+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:59:23.338+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:59:23.338+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:59:23.347+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:59:23.345+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:59:23.348+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:59:23.366+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.040 seconds
[2024-05-10T19:59:53.493+0000] {processor.py:161} INFO - Started process (PID=72) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:59:53.494+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T19:59:53.497+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:59:53.496+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:59:53.504+0000] {logging_mixin.py:188} INFO - [2024-05-10T19:59:53.503+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T19:59:53.504+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T19:59:53.524+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.039 seconds
[2024-05-10T20:00:23.642+0000] {processor.py:161} INFO - Started process (PID=73) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:00:23.643+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T20:00:23.646+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:00:23.645+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:00:23.653+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:00:23.652+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T20:00:23.653+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:00:23.674+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.040 seconds
[2024-05-10T20:00:53.788+0000] {processor.py:161} INFO - Started process (PID=74) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:00:53.789+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T20:00:53.791+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:00:53.791+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:00:53.799+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:00:53.798+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T20:00:53.799+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:00:53.816+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.035 seconds
[2024-05-10T20:01:23.928+0000] {processor.py:161} INFO - Started process (PID=75) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:01:23.929+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T20:01:23.931+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:01:23.931+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:01:23.938+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:01:23.937+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T20:01:23.939+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:01:23.958+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.036 seconds
[2024-05-10T20:01:54.079+0000] {processor.py:161} INFO - Started process (PID=76) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:01:54.081+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T20:01:54.083+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:01:54.083+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:01:54.092+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:01:54.091+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T20:01:54.093+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:01:54.119+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.050 seconds
[2024-05-10T20:02:24.213+0000] {processor.py:161} INFO - Started process (PID=77) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:02:24.214+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T20:02:24.217+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:02:24.216+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:02:24.226+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:02:24.224+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T20:02:24.226+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:02:24.246+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.039 seconds
[2024-05-10T20:02:54.380+0000] {processor.py:161} INFO - Started process (PID=78) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:02:54.383+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T20:02:54.390+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:02:54.389+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:02:54.406+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:02:54.404+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T20:02:54.409+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:02:54.459+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.087 seconds
[2024-05-10T20:03:24.535+0000] {processor.py:161} INFO - Started process (PID=79) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:03:24.536+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T20:03:24.539+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:03:24.538+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:03:24.554+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:03:24.552+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T20:03:24.554+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:03:24.581+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.055 seconds
[2024-05-10T20:03:55.299+0000] {processor.py:161} INFO - Started process (PID=29) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:03:55.300+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T20:03:55.308+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:03:55.307+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:03:55.325+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:03:55.321+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T20:03:55.326+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:03:55.354+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.071 seconds
[2024-05-10T20:04:25.484+0000] {processor.py:161} INFO - Started process (PID=30) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:04:25.596+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T20:04:25.599+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:04:25.599+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:04:25.608+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:04:25.607+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T20:04:25.609+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:04:25.695+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.220 seconds
[2024-05-10T20:04:55.871+0000] {processor.py:161} INFO - Started process (PID=31) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:04:55.886+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T20:04:55.889+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:04:55.888+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:04:55.898+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:04:55.897+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T20:04:55.898+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:04:56.025+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.163 seconds
[2024-05-10T20:05:26.071+0000] {processor.py:161} INFO - Started process (PID=32) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:05:26.072+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T20:05:26.075+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:05:26.074+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:05:26.083+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:05:26.082+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T20:05:26.083+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:05:26.384+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.322 seconds
[2024-05-10T20:05:56.519+0000] {processor.py:161} INFO - Started process (PID=33) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:05:56.541+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T20:05:56.544+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:05:56.543+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:05:56.552+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:05:56.551+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T20:05:56.552+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:05:56.673+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.164 seconds
[2024-05-10T20:06:26.806+0000] {processor.py:161} INFO - Started process (PID=34) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:06:26.829+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T20:06:26.831+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:06:26.831+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:06:26.839+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:06:26.838+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T20:06:26.840+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:06:27.032+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.232 seconds
[2024-05-10T20:06:57.155+0000] {processor.py:161} INFO - Started process (PID=35) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:06:57.156+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T20:06:57.159+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:06:57.159+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:06:57.167+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:06:57.166+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T20:06:57.167+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:06:57.763+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.614 seconds
[2024-05-10T20:07:27.893+0000] {processor.py:161} INFO - Started process (PID=36) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:07:27.948+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T20:07:27.950+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:07:27.949+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:07:27.957+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:07:27.956+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T20:07:27.958+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:07:35.614+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 7.727 seconds
[2024-05-10T20:08:05.755+0000] {processor.py:161} INFO - Started process (PID=37) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:08:05.756+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T20:08:05.759+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:08:05.759+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:08:05.772+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:08:05.770+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T20:08:05.772+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:08:05.801+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.057 seconds
[2024-05-10T20:08:36.401+0000] {processor.py:161} INFO - Started process (PID=38) to work on /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:08:36.409+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2024-05-10T20:08:36.417+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:08:36.411+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:08:36.441+0000] {logging_mixin.py:188} INFO - [2024-05-10T20:08:36.438+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2024-05-10T20:08:36.442+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2024-05-10T20:08:36.490+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.097 seconds
